TestAssets/TestProjects/MSBuildTestApp/Program.cs

```diff
@@ -9,6 +9,10 @@ public class Program
     {
         public static void Main(string[] args)
         {
+          if (args.Length > 0)
+            {
```

Comment:
NIT: weird tabing here. The brackets should be in the same line as the if.

NIT: empty line after closing curly bracket.